packages/@uppy/companion-client/src/RequestClient.js

```diff
@@ -41,17 +42,25 @@ async function handleJSONResponse (res) {
   }
 
   let errMsg = `Failed request with status: ${res.status}. ${res.statusText}`
+  let errData
   try {
-    const errData = await res.json()
+    errData = await res.json()
 
     errMsg = errData.message ? `${errMsg} message: ${errData.message}` : errMsg
     errMsg = errData.requestId
       ? `${errMsg} request-Id: ${errData.requestId}`
       : errMsg
```

Comment:
This is now unused I think
```suggestion
```

Reply:
oops right i must have messed up in the merge conflict